### ISSUE TITLE
Add topics to Github project page to make project more discoverable 

### ISSUE THREAD

**Issue by Pentaphon** [transmission/transmission#6150]
### What is the issue?

Github has a topics feature to categorize projects and help people find similar projects and the Transmission project has not utilized this simple feature to make the project more discoverable. 

For example, when you click https://github.com/topics/bittorrent-client Transmission is missing from the list of all the torrent client project pages on Github!

The people with admin access to this project page should add the following topics and any others they might find relevant:

[c-plus-plus](https://github.com/topics/c-plus-plus)
[crossplatform](https://github.com/topics/crossplatform)
[torrent](https://github.com/topics/torrent)
[bittorrent](https://github.com/topics/bittorrent)
[bittorrent-client](https://github.com/topics/bittorrent-client)
[torrent-client](https://github.com/topics/torrent-client)

This way, we can attract more contributors using Github's browse and search features, especially those looking for C++ projects to work on.  Every tiny bit helps, and not everybody knows about Transmission or Transmission's presence on Github.

### Which application of Transmission?

None

### Which version of Transmission?

None

**Comment by nevack** [comment]
Would add [macOS](https://github.com/topics/macos) and other OSes as well 😄

**Comment by Pentaphon** [comment]
> Would add [macOS](https://github.com/topics/macos) and other OSes as well

Yes, if anybody with permissions to do this is reading this, please use this guide to add topics to the project. 

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics

**Comment by ckerr** [comment]
It doesn't really look like anyone else is using Topics.

When I search for 'bittorrent' in them it just gives a couple of spam projects, nothing serious like Transmission or qBT or libtorrent or rakshasa etc.

**Comment by Pentaphon** [comment]
> When I search for 'bittorrent' in them it just gives a couple of spam projects, nothing serious like Transmission or qBT or libtorrent or rakshasa etc.

I beg to differ. Topics can certainly help people discover Transmission from the following topics used by qBT. 

<img width="240" height="174" alt="Image" src="https://github.com/user-attachments/assets/19f69d56-95ab-4e20-a93b-78ab38fcdcf3" />

**Comment by ckerr** [comment]
@Pentaphon did you try clicking on those topics? 4 of the 6 are dead links.

BitTorrent seems to not be a valid topic. https://github.com/search?q=bittorrent&type=topics

**Comment by Pentaphon** [comment]
>did you try clicking on those topics? 4 of the 6 are dead links.

Point taken. I did not click them since I started this issue. Can you still do the 2 that are still active? I know it's a small thing but it could still help with discovery for C++ and crossplatform projects.
